Community Toolbox/objects/ctrl_ArrayUtilities/Create_0.gml

```diff
@@ -1,35 +1,97 @@
 title = "Array Utilities";
-description = "Calculate the maximum, minimum, median, mean, and sum values of an array of numbers.";
+description = "Calculate the maximum, minimum, median, mean, and sum, and get/pop random values, of an array.";
```

Comment:
Nit: There's two "and"s in this sentence now.

Reply:
Fixed, and also tweaked the demo in general (in particular changing the positioning of offset/length buttons to make more space for the array items.